egov-persister/department-entity-persister.yml

```diff
@@ -0,0 +1,93 @@
+serviceMaps:
+  serviceName: department-entity-services
+  mappings:
+    - version: 1.0
+      description: Save departemnt entity details in tables
+      fromTopic: save-dept-entity-application
+      isTransaction: true
+      queryMaps:
+
+        - query: INSERT INTO department_entity(id, tenant_id, department_id, code, name, hierarchy_level, children, created_by, last_modified_by, created_time, last_modified_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
+          basePath: departmentEntity.*
+          jsonMaps:
+            - jsonPath: $.departmentEntity.*.id
+
+            - jsonPath: $.departmentEntity.*.tenantId
+
+            - jsonPath: $.departmentEntity.*.departmentId
+
+            - jsonPath: $.departmentEntity.*.code
+
+            - jsonPath: $.departmentEntity.*.name
+
+            - jsonPath: $.departmentEntity.*.hierarchyLevel
+
+            - jsonPath: $.departmentEntity.*.children
+            
+            - jsonPath: $.departmentEntity.*.createdBy
+
+            - jsonPath: $.departmentEntity.*.lastModifiedBy
+
+            - jsonPath: $.departmentEntity.*.createdTime
+
+            - jsonPath: $.departmentEntity.*.lastModifiedTime
+
+    - version: 1.0
+      description: Update departemnt entity details in tables
+      fromTopic: update-dept-entity-application
+      isTransaction: true
+      queryMaps:
+        - query: UPDATE department_entity SET tenant_id = ?,department_id = ?,code = ?,name = ?, hierarchy_level = ?, children = ?, created_by = ?, last_modified_by = ?, created_time = ?, last_modified_time = ? WHERE id=?;

```

Comment:
why are we allowing fields like createdtime, createdby for update? For update use case it only makes sense to update lastmodifiedby and lastmodifiedtime. Also, please check which fields were updatable back when this service was using mongo database and allow only those fields.